chcemvediet/apps/inforequests/tests/test_cron.py

```diff
@@ -185,38 +186,32 @@ def test_reminder_is_sent_if_newest_undecided_email_newer_than_last_reminder(sel
         message_set = self._call_cron_job()
         self.assertTrue(message_set.exists())
 
-    @unittest.skip('FIXME')
     def test_inforequest_is_skipped_if_exception_raised_while_checking_it(self):
         timewarp.jump(local_datetime_from_local(u'2010-10-05 10:33:00'))
-        inforequests = [self._create_inforequest() for i in range(3)]
-        emails = [self._create_inforequest_email(inforequest=ir) for ir in inforequests]
+        inforequest = self._create_inforequest()
+        email = self._create_inforequest_email(inforequest=inforequest)
```

Comment:
Preco namiesto troch inforequestov si nechal iba jeden. Ten test by mal otestovat to, ze sa preskonci **iba** ten inforequest, ktory raisne exception. Ale ostatne inforequesty sa spracuju normalne.

Rovnako v teste nizsie.

Reply:
Lebo ked som mockol funckiu `workdays.between` uz v __init__ tak to nebolo mozne otestovat viacero inforequestov naraz, aby niektore presli a niektore nie. Pridanim funkcie `nop` sato vsak vyriesilo.

Reply:
:ok: